src/Analyzers/CSharp/Tests/MakeDeclarationPartial/MakeDeclarationPartialTests.cs

```diff
@@ -0,0 +1,257 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp.MakeDeclarationPartial;
+using Microsoft.CodeAnalysis.Editor.UnitTests.CodeActions;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.CodeAnalysis.Testing;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.CSharp.UnitTests.MakeDeclarationPartial
+{
+    using VerifyCS = CSharpCodeFixVerifier<
+        EmptyDiagnosticAnalyzer,
+        CSharpMakeDeclarationPartialCodeFixProvider>;
+
+    [Trait(Traits.Feature, Traits.Features.CodeActionsMakeDeclarationPartial)]
+    public sealed class MakeDeclarationPartialTests
```

Comment:
I don't see a test where there are two different classes, so that we properly test fix all support.

```csharp
public class C1 { }
public partial class C1 { }
public class C2 { }
public partial class C2 { }
```

Reply:
I didn't initially plan this codefix to have fix all support. This is very niche fix that won't be executed in masses

Reply:
I think supporting fix all would be a single line (overriding `GetFixAllProvider` and return `BatchFixer`). Given the simplicity, I don't see a reason why to not support fix all.

Reply:
As it turned out, fix all is broken is some nested type cases. Considering that the main argument for it was simplicity in the first place, now I don't see valid reasons for having it, it just don't worth the effort. The compiler error appears as you type and you can fix it right away, so "fix all" is not something that will be used.